docs/rendering/modelextensions/transforms.md

```diff
@@ -0,0 +1,67 @@
+Root Transforms
+===============
+
+Adding the `transform` entry at the top level of a model JSON allows specifying a transformation that will be baked into the vertex data of all quads of the model. These transforms apply before the transforms specified in the [blockstate] file of a block model and the [display transforms][displaytransform] of an item model.
+
+The root transforms can be specified in four formats:
+
+1. A string referring to a default value (currently only supports the value `"identity"`)
+2. A raw transformation matrix in the form of a nested JSON array with the last row omitted (3*4 matrix, row major order)
+3. A JSON object containing a singular `matrix` entry with the same format as above
```

Comment:
I would still like an example for these as it not generally clear how this would look:

```js
{
    "transform": [
       [4, 5, 6], // What do these numbers represent?
       // ...
    ]
}
```

Reply:
I've added an example purely to demonstrate the expected structure. At the moment, I can't explain what the individual values in the matrix represent as I am failing to even find a simple matrix that works at all. While searching one, I may have found a severe bug in this implementation (apart from two of the transform formats turning out to not be usable at the moment) but I'll need to confirm that with further tests.